ui/apps/platform/src/Containers/ConfigManagement/Entity/widgets/FailedPoliciesAcrossDeployment.js

```diff
@@ -87,12 +87,12 @@ const FailedPoliciesAcrossDeployment = ({ deploymentID }) => {
                         accessor: 'name',
                     },
                     {
-                        Header: `Enforced`,
+                        Header: `Enforcing`,
                         headerClassName: `w-1/8 ${defaultHeaderClassName}`,
                         className: `w-1/8 ${defaultColumnClassName}`,
                         Cell: ({ original }) => {
                             const { enforcementActions } = original;
```

Comment:
The value of `enforcementActions` is always an array (empty when not enforcing), so this check for truthiness will always be `true`.